IdentityCore/src/cache/accessor/MSIDAccountCredentialCache.m

```diff
@@ -157,14 +158,18 @@ - (nullable MSIDCredentialCacheItem *)getCredential:(nonnull MSIDDefaultCredenti
         NSMutableArray<MSIDAccountCacheItem *> *filteredResults = [NSMutableArray array];
 
         BOOL shouldMatchAccount = !cacheQuery.homeAccountId || !cacheQuery.environment;
+        MSID_LOG_VERBOSE(context, @"(Default account cache) shouldMatchAccount %@", shouldMatchAccount? @"yes" : @"no");
 
         for (MSIDAccountCacheItem *cacheItem in cacheItems)
         {
+            MSID_LOG_VERBOSE(context, @"(Default account cache) cacheItem hashed home acccount ID %@, environment %@", [cacheItem.homeAccountId msidTokenHash], cacheItem.environment);
```

Comment:
Discussed offline and agreed to add caching for cacheItem has as well

Reply:
Addressed